config/src/wsv.rs

```diff
@@ -11,9 +11,7 @@ const DEFAULT_METADATA_LIMITS: MetadataLimits = MetadataLimits::new(2_u32.pow(20
 const DEFAULT_IDENT_LENGTH_LIMITS: LengthLimits = LengthLimits::new(1, 2_u32.pow(7));
 
 /// `WorldStateView` configuration.
-#[derive(
-    Debug, Clone, Copy, PartialEq, Eq, Deserialize, Serialize, Proxy, LoadFromEnv, Documented,
-)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Deserialize, Serialize, Proxy, Documented)]
```

Comment:
You should consider re-exporting `Documented` and `Proxy` from `iroha_config_base`. As a result, you could set up something like either `iroha_config_base::Proxy`, or `config::Proxy`, so that the namespacing disambiguates the `Proxy` and `Documented` terms. 

`ConfigProxy` is also a viable option, but it's usually better to use namespaces, in case someone wants to shorten the naming.

Reply:
You mean bringing these derives up to `iroha_config` level outside of `iroha_config_base`? Or disambiguating the derive and its trait? If it's the latter, they are already namespaced by belonging to `iroha_config_base::derive::Documented` and `iroha_config_base::proxy::Documented` respectively.

Reply:
The former.